### ISSUE TITLE
5.0 is super-slow!

### ISSUE THREAD

**Issue by mitrakov** [jurplel/qView#514]
**Environment:**
- OS: [macOS 10.15.7 Catalina]
- qView version: [5.0]

**Information:**
5.0 release is extremely slow! Not just a startup time, but also listing the files with LEFT and RIGHT arrow keys.

**Comment by RoyBellingan** [comment]
I just downloaded and compiled eee6b693e3694876950bada0d47756803b2aed14

One problem is connected on rescanning the whole folder, each time a picture is advanced, I was using now a massive 2400 picture one, and it was slowing down a good amount. going to a 10 file folder, it helped a good deal.
I think using regex just to check the file extension is a bit overkill...

Also, but I have NO IDEA how to do that, I think adding a check somehow if jemalloc is present to use it... 

On a folder with 600 picture the function void QVImageCore::updateFolderInfo() was responsibile (on a debug build) for 70% of the time, because all the internal check done by intermediate lib are just expensive...

BTW I saw that this commit changed from just checking the filename extension to the regex, I will recheck how long this old one took and report some info.

There are a few call also to sort the list, which is expensive in itself.

I will try to play around, as code is really written in a good way, and I like this program.
